desktop/libs/notebook/src/notebook/connectors/base.py

```diff
@@ -406,6 +407,9 @@ def patch_snippet_for_connector(snippet):
   Connector backward compatibility switcher.
   # TODO Connector unification
   """
+  if snippet['type'] == 'hive-compute' or snippet['type'] == 'impala-compute':
+    # No patching is needed
+    return
```

Comment:
return `None`?

Reply:
The function doesn't return anything even otherwise. Do we still need to explicitly return a `None`?

Reply:
Nothing set in stone, its just a nit. IIRC I read sometime back in PEP docs to explicitly set None for return values in such cases.